pandas/core/series.py

```diff
@@ -2281,36 +2285,62 @@ def _binop(self, other, func, level=None, fill_value=None):
 
     def combine(self, other, func, fill_value=None):
         """
-        Perform elementwise binary operation on two Series using given function
-        with optional fill value when an index is missing from one Series or
-        the other
+        Combine the Series with a Series or scalar according to `func`.
+
+        Combine the Series and `other` using `func` to perform elementwise
+        selection for combined Series.
+        `fill_value` is assumed when value is missing at some index
+        from one of the two objects being combined.
 
         Parameters
         ----------
-        other : Series or scalar value
+        other : Series or scalar
+            The value(s) to be combined with the `Series`.
         func : function
-            Function that takes two scalars as inputs and return a scalar
-        fill_value : scalar value
-            The default specifies to use the appropriate NaN value for
-            the underlying dtype of the Series
+            Function that takes two scalars as inputs and returns an element.
+        fill_value : scalar, optional
+            The value to assume when an index is missing from
+            one Series or the other. The default specifies to use the
+            appropriate NaN value for the underlying dtype of the Series.
 
         Returns
         -------
-        result : Series
+        Series
+            The result of combining the Series with the other object.
+
+        See Also
+        --------
+        Series.combine_first : Combine Series values, choosing the calling
+                               Series' values first.
 
         Examples
         --------
         >>> s1 = pd.Series([1, 2])
-        >>> s2 = pd.Series([0, 3])
-        >>> s1.combine(s2, lambda x1, x2: x1 if x1 < x2 else x2)
-        0    0
-        1    2
+        >>> s2 = pd.Series([0, 3, 4])
+        >>> s1.combine(s2, lambda x1, x2: x1 if x1 > x2 else x2)
+        0    1
+        1    3
+        2    4
+        dtype: int64
+
+        >>> arms = pd.Series({'dog':2,'cat': 2,'mouse': 2})
+        >>> legs = pd.Series({'dog':2,'cat': 2})
+        >>> arms
+        dog      2
+        cat      2
+        mouse    2
+        dtype: int64
+        >>> legs
+        dog    2
+        cat    2
+        dtype: int64
+        >>> limbs = arms.combine(legs, lambda x1,x2: x1+x2, fill_value=0)
```

Comment:
Do not assign, show the result directly.

Show the example without `fill_value` first, and then add `fill_value=whatever` to show the difference.

Would be nice to avoid `lambda`, if `max` or `min` can make sense, better use those.

There are pep8 errors, `scripts/validate_scripts.py` reports them now, make sure there are no validation errors before pushing, we built that script to avoid reviewing things that a script can tell you.